doc/nrf/gs_modifying.rst

```diff
@@ -198,25 +306,33 @@ Creating build type files
 
 To create custom build type files for your application instead of using a single :file:`prj.conf` file, complete the following steps:
 
-1. During :ref:`application development <zephyr:application>`, follow the procedure for creating the application until after the step where you create the :file:`CMakeLists.txt` file.
-#. In the :file:`CMakeLists.txt` file, define the file name pattern for configuration files.
-   For example::
-
-    set(CONF_FILE "app_${CMAKE_BUILD_TYPE}.conf")
+1. Create a new :file:`prj_<buildtype>.conf` in the main application folder next to the :file:`prj.conf` file, with *<buildtype>* in the file name corresponding to your build type name.
+   For example, :file:`prj_release.conf`.
+#. Edit the new file to set the desired Kconfig configuration options.
+#. Save the changes.
 
-   In this define, ``CMAKE_BUILD_TYPE`` will be used for selecting the build type.
-#. Optionally, include an if statement that checks for the presence of the selected build type configuration files.
-   For an example, see :file:`applications/nrf_desktop/CMakeLists.txt`.
-#. Continue the application creation procedure by setting the Kconfig configuration options.
-#. Save the :file:`.conf` file in the application directory with a name that matches the file name pattern defined in CMakeLists.
-   For example, :file:`app_ZRelease.conf`.
-   In this file name, ``ZRelease`` is the build type name.
+In case there are other configuration files that you would like to modify, such as Partition Manager :ref:`static configuration <ug_pm_static>` or any child image configuration, you need to create a separate configuration file for them using the same naming pattern.
+For example, using the example above, the file for the Partition Manager static configuration would be called :file:`pm_static_release.yaml`.
 
-You can now select build types in SES or from command line.
+The Devicetree configuration does not follow this configuration pattern, as it is defined per board, not per build type.
+In other words, the :file:`app.overlay` is common for all configurations.
```

Comment:
Delete the two lines above and replace with:

The Device tree configuration is not affected by the build type.

Full stop.

Device tree should be respeciting it, the only reason it is not is because we were not able to get this through review.

Reply:
Done.